node/cli/src/lib.rs

```diff
@@ -100,19 +99,25 @@ pub fn get_service_configs() -> (NetworkConfig, ClientConfig) {
                 .takes_value(true),
         ).arg(
             Arg::with_name("account_id")
-            .value_name("ACCOUNT_ID")
-            .help("Set the account id of the node")
-            .takes_value(true)
-                // TODO(#282): Remove default account id from here.
-            .default_value("alice.near")
+                .value_name("ACCOUNT_ID")
+                .help("Set the account id of the node")
+                .takes_value(true)
+                    // TODO(#282): Remove default account id from here.
+                .default_value("alice.near")
         ).arg(
             Arg::with_name("public_key")
-              .short("k")
-              .long("public-key")
-              .value_name("PUBLIC_KEY")
-              .help("Sets public key to sign with, \
-                         can be omitted with 1 file in keystore")
-              .takes_value(true)
+                .short("k")
+                .long("public-key")
+                .value_name("PUBLIC_KEY")
+                .help("Sets public key to sign with, \
+                           can be omitted with 1 file in keystore")
+                .takes_value(true)
+        ).arg(
+            Arg::with_name("block_period")
```

Comment:
i have been prefixing flags that are only used in testing with `test` like this https://github.com/nearprotocol/nearcore/blob/3e9f5291f20f2367b31a9e56d1b9b0cf4da0e73b/node/cli/src/lib.rs#L79-L87

i don't think it's the right solution, but think it's good enough for now. eventually we should throw an error if this flag is set and you do not specify that it's devnet, because this will never be used. we can do this with Clap and conditional validation also.

Reply:
i think it's also good to indicate the unit of value (milliseconds)

Reply:
I am not entirely convinced that this is solely for tests. For devnet, we might want to change the block production speed as well.

Reply:
i misspoke. the other flags are also necessary for devnet and testnet.

Reply:
Also it is the case for now that if the flag is set, only devnet will use the flag.

Reply:
yep.. one of the points is that it is confusing behavior to have flags that are silently ignored in certain cases

Reply:
also.. with the conditional you moved to.. this flag will trigger devnet consensus behavior which is unexpected

Reply:
I don't think so. https://github.com/nearprotocol/nearcore/blob/bb1ba6b38bb2e9e6c7be4eac78fd631c010221f9/node/cli/src/lib.rs#L192

Reply:
i included this conversation in https://github.com/nearprotocol/nearcore/issues/418 so if we don't want to make any changes now, this can be addressed later

Reply:
I did the change in 0f21c05